CMakeLists.txt

```diff
@@ -30,9 +30,8 @@ target_link_libraries(binja-pattern
     binaryninjaapi fmt mem yaml-cpp Zydis)
 
 set_target_properties(binja-pattern PROPERTIES
-    CXX_STANDARD 11
+    CXX_STANDARD 17
     CXX_STANDARD_REQUIRED ON)
 
-binja_install_plugin(binja-pattern)
+bn_install_plugin(binja-pattern)
 
-install(FILES "python/binarypattern.py" DESTINATION ${BINJA_PLUGINS_DIR})
```

Comment:
I'd like to keep this install if possible. Looks like they changed the var to `BN_USER_PLUGINS_DIR`.

https://github.com/Vector35/binaryninja-api/blob/dev/cmake/FindBinaryNinjaCore.cmake#L73

Reply:
Afaik, bn_install_plugin already performs the install. https://github.com/Vector35/binaryninja-api/blob/dev/CMakeLists.txt#L68

Did I miss anything?

Reply:
bn_install_plugin only installs the native plugin binaries, not that python script.

Reply:
I see, in this case we also need to load BinaryNinjaCore for CMake, otherwise the path will not be available.
I force-pushed the required changes.